modules/creating-cluster-with-aws-kms-key.adoc

```diff
@@ -0,0 +1,198 @@
+// Module included in the following assemblies:
+//
+// * monitoring/enabling-monitoring-for-user-defined-projects.adoc
+
+:_content-type: PROCEDURE
+[id="creating-cluster-with-aws-kms-key"]
+= Creating a ROSA cluster using a custom AWS KMS key
+
+You can create a {product-title} (ROSA) cluster with a customer-provided KMS key that is used to encrypt both node root volumes and persistent volumes for mounted EBS claims.
+
+.Procedure
+
+. Create a custom AWS customer-managed KMS key by running the following command:
++
+[source,terminal]
+----
+$ KMS_ARN=$(aws kms create-key --region $AWS_REGION --description 'Custom ROSA Encryption Key' --tags TagKey=red-hat,TagValue=true --query KeyMetadata.Arn --output text)
```

Comment:
Can we add some kind of callout or `Note` admonition to mention that `--tags TagKey=red-hat,TagValue=true` is an important and required piece of _any_ customer KMS key being provided by the customer?